player/lua/osc.lua

```diff
@@ -1727,6 +1743,12 @@ function validate_user_opts()
                 user_opts.windowcontrols_alignment .. "\". Ignoring.")
         user_opts.windowcontrols_alignment = "right"
     end
+
+    for _,color in pairs({user_opts.background_color, user_opts.top_buttons_color, user_opts.small_buttonsL_color, user_opts.small_buttonsR_color, user_opts.buttons_color, user_opts.title_color, user_opts.timecode_color, user_opts.time_pos_color, user_opts.held_element_color, user_opts.time_pos_color}) do
```

Comment:
Missing a space in `,color` and has `time_pos_color` twice instead of `time_pos_outline_color`.

Reply:
Between `,` and `color`?